CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+8.4.0
+----
+* Fix embedded app session management in Safari 12.1
```

Comment:
might be nice to include this as well: https://github.com/Shopify/shopify_app/pull/667

Reply:
Ah I should've added this to the CHANGELOG, we need a PR template in here.